components/nimbus/ios/Nimbus/Nimbus.swift

```diff
@@ -140,9 +140,9 @@ extension Nimbus: FeaturesInterface {
     internal func getFeatureConfigVariablesJson(featureId: String) -> [String: Any]? {
         do {
             if let string = try nimbusClient.getFeatureConfigVariables(featureId: featureId),
```

Comment:
nit: Since the `else` case of the `if let` here only returns `nil`, I think `guard let` could be appropriate here